media-video/showtime/showtime-49.0.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{11..14} )
+PYTHON_REQ_USE="threads(+)"
+
+inherit gnome.org gnome2-utils meson xdg python-single-r1 python-utils-r1
+
+DESCRIPTION="Video player for GNOME"
+HOMEPAGE="https://apps.gnome.org/Showtime/ https://gitlab.gnome.org/GNOME/showtime/"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="+python"
+
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+"
+
+DEPEND="
+	>=x11-libs/gtk+-3.22.0:3[introspection]
+	>=dev-libs/gobject-introspection-1.54:=
+	>=dev-util/blueprint-compiler-0.17.0
+	>=media-plugins/gst-plugin-gtk4-0.13.4:1.0
+	>=dev-python/pygobject-2.90.3:3
```

Comment:
This needs `[${PYTHON_USEDEP}]`, see https://devmanual.gentoo.org/eclass-reference/python-single-r1.eclass/index.html#lbAG again. It is imported by showtime, so must be for the same python version.